File: tarea-1/Item.java
public class Item {
    private int id;                
    private String tipo;
    private String nombre;
    private int cantidad;
        
    public Item(int id, String tipo, String nombre, int cantidad) {
        this.id = id;
        this.tipo = tipo;
        this.nombre = nombre;
        this.cantidad = cantidad;
    }
    public int getId() {return id;}

    public String getTipo() {return tipo;}

    public String getNombre() {return nombre;}

    public int getCant() {return cantidad;}

    public void setCant(int newCant) {
        cantidad = newCant;
    }
    
    public void mostrarItem() {
        System.out.println(this.id + ": " + this.nombre + ", cantidad " + this.cantidad);
    }
}

File: tarea-3/etapa-1/Item.cpp
#include <iostream>
#include <string>
#include "../headers/Item.h"

Item::Item(int i, std::string t, std::string n, int c) {
	id = i;
	tipo = t;
	nombre = n;
	cantidad = c;
}
int Item::getId() {
	return id;
}
void Item::usar(Mascota &mascota) {} 
std::string Item::getTipo() {
	return tipo;
}
std::string Item::getNombre() {
	return nombre;
}
int Item::getCant() {
	return cantidad;
}
void Item::setCant(int newCant) {
	cantidad = newCant;
}
void Item::mostrarItem() {
	std::cout << id << ": " << nombre << ", cantidad " << cantidad <<  std::endl;
};
